.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+First check your title. If this pull request has incomplete or unstable code prefix the title with [WIP]. Often people will give you feedback anyway, but if you are specifically requesting feedback prefix [Feedback Requested] to the title.
+
+NOTE: This template is not enforced. You are encouraged to include all of the detail below, but if anything doesn't fit this particular pull request feel free to delete it!
+
+### The issue this fixes
+
+Briefly describe the issue(s) addressed by this pull request. If there are any relevant issues or pull requests on Github reference their #numbers here.
+
+Fixes # 
+Closes # 
+
+### What this PR does
+
+Describe _how_ you have fixed the issue(s).
+If there are any UI or sprite changes, please include an image showing the new element(s).
```

Comment:
Could we add a note saying something like that "After doing significant changes to the original design and when you are ready for a new review of the PR, add a new image with the new look"?

For animations you can include a gif. Should this be required?